Script.js

```diff
@@ -22,3 +22,180 @@ const move = document.getElementsByClassName('move');
+  tech: ['HTML', 'CSS', 'JavaScript'],
+  live: 'https://andream2429.github.io/Portfolio_setup_and_mobile_first/',
+  source: 'https://github.com/AndreaM2429/Portfolio_setup_and_mobile_first/',
+};
+
+function loadwindos(wprk, index) {
+  const index1 = 7;
+  const index4 = 4;
+  const index5 = 5;
+  const index6 = 6;
+
+  const unlis = document.createElement('ul');
+  const divtul = document.createElement('div');
+  divtul.className = 'padingul';
+  const lilis1 = document.createElement('li');
+  lilis1.className = 'li1';
+  lilis1.appendChild(document.createTextNode(projects.list[0]));
+  unlis.appendChild(lilis1);
+  const liscont = document.createElement('li');
+  const counter = document.createElement('img');
+  counter.src = projects.imag[index1];
+  liscont.appendChild(counter);
+  unlis.appendChild(liscont);
+  const lilissec = document.createElement('li');
+  lilissec.className = 'lisec';
+  lilissec.appendChild(document.createTextNode(projects.list[1]));
+  unlis.appendChild(lilissec);
+  const liscont2 = document.createElement('li');
+  const counter2 = document.createElement('img');
+  counter2.src = projects.imag[index1];
+  liscont2.appendChild(counter2);
+  unlis.appendChild(liscont2);
+  const lilissec2 = document.createElement('li');
+  lilissec2.className = 'lisec';
+  lilissec2.appendChild(document.createTextNode(projects.list[2]));
+  unlis.appendChild(lilissec2);
+  divtul.appendChild(unlis);
+
+  const divtitins = document.createElement('div');
+  divtitins.className = 'pop padingul';
+  const tit = document.createElement('h4');
+  tit.className = 'c1dp mobim';
+  tit.textContent = projects.name[index];
+  const titdesk = document.createElement('h4');
+  titdesk.className = 'c1dp deskim';
+  titdesk.textContent = projects.namedesk[index];
+  const closebut = document.createElement('a');
+  closebut.className = 'closepop botonclose efecbot';
+  const imgclose = document.createElement('img');
+  imgclose.src = projects.imag[index4];
+  closebut.append(imgclose);
+  divtitins.appendChild(tit);
+  divtitins.appendChild(titdesk);
+  divtitins.appendChild(closebut);
+
+  const divdesk = document.createElement('div');
+  divdesk.classList = 'flexdesk';
+  const descont = document.createElement('div');
+  descont.className = 'pdes';
+  const desc = document.createElement('p');
+  desc.textContent = projects.desciption;
+  desc.className = 'ca1p padingul';
+  const desc2 = document.createElement('p');
+  desc2.textContent = projects.desciption;
+  desc2.className = 'ca1p padingul deskim';
+
+  descont.appendChild(desc);
+  descont.appendChild(desc2);
+  divdesk.appendChild(descont);
+
+  const divtagbut = document.createElement('div');
+  divtagbut.className = 'columdesk';
+  const divtags = document.createElement('div');
+  divtags.className = 'tags';
+  const ultags = document.createElement('ul');
+  ultags.className = 'padingul';
+  for (let i = 0; i < 2; i += 1) {
```

Comment:
Nice job implementing the functionalities of the popup 👍. Let's make some improvements by making our project match the given design👍.

- Let's try to make the popup responsive. on mobile, it is impossible to see the close button and the full popup, so as to close the popup because the popup doesn't fit the screen and it is impossible to scroll. kindly fix this and ensure you stick to the design please as this can lead to a bad user experience👍.
<img width="375" alt="image" src="https://github.com/AndreaM2429/Portfolio_setup_and_mobile_first/assets/64643995/0ede47a7-f758-46e0-8944-ec963928cb72">


for the desktop page, let's try to ensure our project matches the given design😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="500" alt="image" src="https://github.com/AndreaM2429/Portfolio_setup_and_mobile_first/assets/64643995/8c930827-9687-480e-943e-a2c1bebe0c55">|<img width="379" alt="image" src="https://github.com/AndreaM2429/Portfolio_setup_and_mobile_first/assets/64643995/265adb0c-e1e7-4895-86bb-350bc1af5f28">|


Kindly take note of the spacings and position of each elements and make the necessary changes as we a required to stick to the given design in this milestone👍.